### PR TITLE
feat(react-media): Extract media component from Pivotal UI

### PR DESCRIPTION
**Pull request by stubbornella**

### CODE REVIEW

**Comment by stubbornella** [comment]
@charleshansen - tag, you're it!

File: dist/media/package.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    {
+      "name": "vinsonchuong",
+      "email": "vinsonchuong@gmail.com"
+    }
+  ],
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/pivotal-cf/pivotal-ui-react/issues"
+  },
+  "homepage": "https://github.com/pivotal-cf/pivotal-ui-react",
```
Comment (charleshansen) [inline dist/media/package.json]: homepage should point to actual component

File: spec/media/media_spec.js [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+      expect('.media .media-body').toContainText('fop');
+    });
+
+    it('passes the provided className onto the .media element', function() {
+      expect('.media').toHaveClass('media-class');
+    });
+  });
+
+  describe('when left image src is set', function() {
+    describe('when left image src is set', function() {
```
Comment (charleshansen) [inline spec/media/media_spec.js]: maybe change second describe to 'when padding is normal' or something less confusing.

File: spec/media/media_spec.js [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+    });
+  });
+
+  describe('when image alignment is set to bottom', function() {
+    beforeEach(function() {
+      var image = (<img src="http://placehold.it/20x20"/>);
+      React.render((<Media leftImage={image} vAlign="bottom">fop</Media>), root);
+    });
+
+    it('displays the media-middle class', function() {
```
Comment (charleshansen) [inline spec/media/media_spec.js]: should be media-bottom in the it?

File: dist/media/package.json (hunk 1 above)
Comment (stubbornella) [inline dist/media/package.json]: Nice catch, thanks!

File: spec/media/media_spec.js (hunk 2 above)
Comment (stubbornella) [inline spec/media/media_spec.js]: ok, fixed

**Pull request merged**